packages/jetpack-blocks/package.json

```diff
@@ -50,7 +54,7 @@
   },
   "scripts": {
     "clean": "npx rimraf dist",
-    "build": "node bin/build.js",
+    "build": "node $NODE_ARGS ./node_modules/webpack/bin/webpack.js --config ./webpack.config.js",
```

Comment:
:heart:

Reply:
`webpack-cli` should be a `devDependency`, it's bin will be available, then we should be able to do this:

```suggestion
    "build": "webpack --config ./webpack.config.js",
```

Reply:
Even :heart: er

Reply:
https://github.com/Automattic/wp-calypso/pull/31503/commits/ad8acb90f3239b993ad79804fe3409ff5e2ae98f